codecov.yml

```diff
@@ -5,4 +5,7 @@
 fixes:
 - "sourmash::src/sourmash"
 ignores:
-- "src/core/src/ffi"
\ No newline at end of file
+- "src/core/src/ffi"
+
+codecov: 
+ token: 66273971-681c-44a5-b93a-f60249a2a70c
```

Comment:
Even tho the only consequence is someone else uploading garbage reports, maybe add this as a secret?